pkg/cmd/pr/view/view_test.go

```diff
@@ -175,6 +176,9 @@ func runCommand(rt http.RoundTripper, branch string, isTTY bool, cli string) (*t
 	factory := &cmdutil.Factory{
 		IOStreams: ios,
 		Browser:   browser,
+		HttpClient: func() (*http.Client, error) {
+			return &http.Client{Transport: rt}, nil
```

Comment:
Maybe not necessary, but assume this was supposed to be piped through, the variable was unused.